README.md

```diff
@@ -1,8 +1,118 @@
+| 06 Singleton Session Bean              |     N      |      N      |            |  NA   |       NA        | Also tried EE5 version of source code with GlassFish 05, no luck.                                                                                                                                                                                         |
+| 07 Servlet Example                     |     Y      |      Y      |            |  NA   |       NA        |
+| 08 Dependency Lookup                   |     Y      |      Y      |     Y      |  NA   |       NA        | This Code was not an InitialContext example but instead is EJB Context like Example 09. I rewrote this use the InitailContext approach to match book. <br/>WildFly made it easy to find correct context string                                            |
+| 09 EJB Lookup                          |     N      |      Y      |     Y      |   NA  |       NA        | Code is originally the same as Example 08 , except for some HTMl text. Seems to be redundant.                                                                                                                                                             |                                                                                                                                                         |
+| 10 Field Injection                     |     Y      |      Y      |     Y      |   NA  |       NA        |                                                                                                                                                                                                                                                           |
+| 11 Setter Injection                    |     Y      |      Y      |            |   NA  |       NA        |                                                                                                                                                                                                                                                           |
+
+
+
+* Chapter 4:
+* Chapter 5:
+* Chapter 6:
+* Chapter 7:
+* Chapter 8:
+* Chapter 9:
+* Chapter 10:
+* Chapter 11:
+* Chapter 12:
+* Chapter 13:
+* Chapter 14:
+* Chapter 15:
+
+TomcatEE 9 = TomEE 9 plus
+WildFly 6 = Version 26.1.2
+
+# Building
+
+To Build targeting Derby Db
+
+    mvn clean package
+
+There are other build options described [later](#other-build-options)
+
+# IDE Usage
+Import the pom.xml into your IDE.
+
+
+# Goal: It Just works
+To provide a 'it just works' experience for myself that others should be able to use.
+This means:
+* Required libraries will be downloaded
```

Comment:
Is the server going to be downloaded as well? If not, what's the point? Most, if not all, required libraries are already available on the local machine, so why to require user (whose connection may not be fast, nor for free) to download  everything twice?

Reply:
Thanks for responding, OSS projects usually have to be prodded to respond to PR requests!  

I used ANT from 2000-2004 so I am rusty but it wasn't apparent how to actually build this code even after downloading the app servers and setting path variables, etc. The JAR files are in various locations, etc the README.md details that, and not all required jars are included.

This JPA code has  been around since at least 2013 in various forms and while you are familiar with it other won't be. 
I feel users should be able to at least build any code base w/o installing or configuring anything beyond java. Every project I have worked in the last 18 years does this and it saves a lot of frustration.  This then allows their navigation through the code using an IDE.